recipe/meta.yaml

```diff
@@ -199,31 +198,16 @@ outputs:
       build:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        # unix only needs these for the strong run exports
+        # only need these for the strong run exports
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('fortran') }}              # [unix]
-        # pythran code needs clang-cl on windows
-        - clang                                  # [win]
-        # WARNING: It's not recommended to use these MinGW compilers with python extensions
-        # numpy.distutils has a complex mechanism to facilitate mixing gfortran and MSVC
-        # https://pav.iki.fi/blog/2017-10-08/pywingfortran.html#building-python-wheels-with-fortran-for-windows
-        - {{ compiler('m2w64_fortran') }}        # [win]
       host:
         - {{ pin_subpackage('scipy', exact=True) }}
-        # need to repeat host deps on windows because we have to do a full rebuild here;
-        # on unix we need a minimum as well so that installation of the wheel will work
+        # need a minimum for (re-)installation of wheel
         - python
         - pip
         - numpy
-        - libblas         # [win]
-        - libcblas        # [win]
-        - liblapack       # [win]
```

Comment:
This line is still needed.

> WARNING (scipy-tests,Lib/site-packages/scipy/integrate/_test_odeint_banded.cp311-win_amd64.pyd): Needed DSO Library/bin/liblapack.dll found in ['liblapack']
WARNING (scipy-tests,Lib/site-packages/scipy/integrate/_test_odeint_banded.cp311-win_amd64.pyd): .. but ['liblapack'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)

Reply:
Sure, conda cannot assume that transitive dependencies keep the ABI. Though in this case `scipy-tests` depends exactly on `scipy`, and so it's not actually a big issue. But OK.

Reply:
Same applies to linux/osx too! 🙃